src/daemon/rpm-ostreed.conf

```diff
@@ -6,3 +6,4 @@
 #AutomaticUpdatePolicy=none
 #IdleExitTimeout=60
 #LockLayering=false
+#InitOverlayFS=false
```

Comment:
Apologies for the nitpick, but could we consistently call this `initoverlayfs` all lowercase in configs, etc.

Maybe for variable names it doesn't really matter, but we don't capitalise initrd/initramfs in the Linux stack in general, would like to follow that.

Reply:
@ericcurtin I will drop this part in the next patch, so no worries.